playbooks/robusta_playbooks/oom_killer.py

```diff
@@ -93,21 +105,36 @@ def pod_oom_killer_enricher(event: PodEvent, params: OomKillParams):
         subject=PodFindingSubject(pod),
     )
 
+    node: Node = Node.readNode(pod.spec.nodeName).obj
     labels = [
         ("Pod", pod.metadata.name),
         ("Namespace", pod.metadata.namespace),
-        ("Node Name", pod.spec.nodeName),
     ]
-    node: Node = Node.readNode(pod.spec.nodeName).obj  # type: ignore
+
     if node:
         allocatable_memory = PodResources.parse_mem(node.status.allocatable.get("memory", "0Mi"))
         capacity_memory = PodResources.parse_mem(node.status.capacity.get("memory", "0Mi"))
         allocated_precent = (capacity_memory - allocatable_memory) * 100 / capacity_memory
-        node_label = (
+
+        node_labels: List[Tuple[str, str]] = []
+        node_labels.extend(labels)
+        node_labels.append(
+            ("Node Name", pod.spec.nodeName)
+        )
+        node_labels.append((
             "Node allocated memory",
             f"{allocated_precent:.2f}% out of {allocatable_memory}MB allocatable",
-        )
-        labels.append(node_label)
+        ))
+
+        if params.node_memory_graph:
+            node_graph = create_node_graph_enrichment(params, node)
```

Comment:
Only the graph should be controlled by the `node_memory_graph` parameter
 The table should always be added

Reply:
done